scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -330,22 +357,23 @@ def main():
 
             if args.verbose:
                 display_count(o_dict, args.indent, args.sort_keys)
-
             if args.save_counts:
                 save_count(o_dict, args.out_path, args.indent, args.sort_keys)
-
+            if args.save_rejected:
+                save_tractogram(outliers_sft, rejected_sft_name)
```

Comment:
`outlier_sft` is empty if `args.save_intermediate_tractograms` is false. Do you mean `initial_sft` here ?